assets/apps/dashboard/src/scss/content/_start.scss

```diff
@@ -48,3 +48,28 @@
 		}
 	}
 }
+
+@import url('https://fonts.cdnfonts.com/css/barlow');
```

Comment:
Is this necessary? Maybe we can use whatever font is available instead.

Reply:
We can aim for a similar look but don't have to get it exactly the same.